Windows SetTooltip invalid memory address

```powershell
PS C:\Users\Simon\go\src\github.com\psidex\EACS> go run .\cmd\test\main.go
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x0 pc=0x6608e6]

goroutine 6 [running]:
github.com/getlantern/systray.(*winTray).setTooltip(0x921fa0, 0x6f6138, 0x1d, 0xc000086058, 0xc0000311b8)
        C:/Users/Simon/go/pkg/mod/github.com/getlantern/systray@v1.0.3/systray_windows.go:226 +0x66
github.com/getlantern/systray.SetTooltip(0x6f6138, 0x1d)
        C:/Users/Simon/go/pkg/mod/github.com/getlantern/systray@v1.0.3/systray_windows.go:852 +0x4c
main.OnReady()
        C:/Users/Simon/go/src/github.com/psidex/EACS/cmd/test/main.go:8 +0x3d
github.com/getlantern/systray.Register.func2(0xc000014180, 0x706528)
        C:/Users/Simon/go/pkg/mod/github.com/getlantern/systray@v1.0.3/systray.go:91 +0x45
created by github.com/getlantern/systray.Register
        C:/Users/Simon/go/pkg/mod/github.com/getlantern/systray@v1.0.3/systray.go:89 +0xe3
exit status 2
```

here is the contents of `main.go`:

```go
package main

import (
	"github.com/getlantern/systray"
)

func OnReady() {
	systray.SetTooltip("This is the tooltip text")
}

func OnExit() {
	// No cleanup needed.
}

func main() {
	systray.Run(OnReady, OnExit)
}

```

I'm still experiencing the same error on 1.0.4 (using the same code in my initial issue)

```powershell
PS C:\Users\Simon\go\src\github.com\psidex\trayTest> go run .\main.go
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x0 pc=0x6608ae]

goroutine 6 [running]:
github.com/getlantern/systray.(*winTray).setTooltip(0x923260, 0x6f5060, 0x18, 0x0, 0x0)
        C:/Users/Simon/go/pkg/mod/github.com/getlantern/systray@v1.0.4/systray_windows.go:238 +0xce
github.com/getlantern/systray.SetTooltip(0x6f5060, 0x18)
        C:/Users/Simon/go/pkg/mod/github.com/getlantern/systray@v1.0.4/systray_windows.go:899 +0x4c
main.OnReady()
        C:/Users/Simon/go/src/github.com/psidex/trayTest/main.go:8 +0x3d
github.com/getlantern/systray.Register.func2(0xc000014180, 0x7075d0)
        C:/Users/Simon/go/pkg/mod/github.com/getlantern/systray@v1.0.4/systray.go:91 +0x45
created by github.com/getlantern/systray.Register
        C:/Users/Simon/go/pkg/mod/github.com/getlantern/systray@v1.0.4/systray.go:89 +0xe3
exit status 2
```

--- a/systray_windows.go
+++ b/systray_windows.go
@@ -254,11 +254,8 @@ func (t *winTray) wndProc(hWnd windows.Handle, message uint32, wParam, lParam ui
 		WM_ENDSESSION = 0x0016
 		WM_CLOSE      = 0x0010
 		WM_DESTROY    = 0x0002
-		WM_CREATE     = 0x0001
 	)
 	switch message {
-	case WM_CREATE:
-		systrayReady()
 	case WM_COMMAND:
 		menuItemId := int32(wParam)
 		// https://docs.microsoft.com/en-us/windows/win32/menurc/wm-command#menus
@@ -771,6 +768,7 @@ func registerSystray() {
 		return
 	}
 
+	systrayReady()
 }
 
 func nativeLoop() {

